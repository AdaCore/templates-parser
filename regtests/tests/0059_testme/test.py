from test_support import *

run('testme', ["testme57.tmplt"])
