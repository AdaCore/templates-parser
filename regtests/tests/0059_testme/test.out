The classes: Class1, Class2
The members: Member1.1, Member1.2
Member2.1, Member2.2, Member2.3

Class1
MISSING OUTPUT:Class1, Class2
->Class1::Member1.1
  ERROR IF PRESENT:Class1, Class2
->Class1::Member1.2
  ERROR IF PRESENT:Class1, Class2

Class2
MISSING OUTPUT:Class1, Class2
->Class2::Member2.1
  ERROR IF PRESENT:Class1, Class2
->Class2::Member2.2
  ERROR IF PRESENT:Class1, Class2
->Class2::Member2.3
  ERROR IF PRESENT:Class1, Class2


